DERMS/ModelBuilder/SummaryCreator/SummaryViewModelContainer.cs

```diff
@@ -7,7 +7,10 @@ namespace ClientUI.SummaryCreator
     public enum ContentType
     {
         AnalogRemotePointSummary,
-        AnalogRemotePointCommandingWindow
+        AnalogRemotePointCommandingWindow,
+        // not sure
```

Comment:
Dobro je, verovatno ćemo refaktorisati, da taj enum ne bude ogroman već da se napravi neki objekat ili nešto slično,